sample-tapd.conf

```diff
@@ -310,13 +310,17 @@
 ; (latency, etc)
 ; prometheus.perfhistograms=false
 
-[experimental.rfq]
+[experimental]
```

Comment:
I'm not sure this change is correct. I think the section headers always need to reflect the full prefix to be valid. The config file parsing library we use is quite particular about these things...

Reply:
I’ve tested it. The experimental RFQ feature doesn’t work with [experimental.rfq] ; it shows a ‘no group found’ warning